src/services/impl/write/TeamService.js

```diff
@@ -0,0 +1,40 @@
+import deipRpc from '@deip/rpc-client';
+import BaseService from './../../base/BaseService';
+import TeamSchema from './../../../schemas/TeamSchema';
+
+
+class TeamService extends BaseService {
+
+  constructor(options = { scoped: true }) { 
+    super(TeamSchema, options);
+  }
+
+  async createTeamRef({
```

Comment:
Please remove postfix `Ref` from the method name. It should be just `createTeam`, `updateTeam`